web/auth/jwt.go

```diff
@@ -18,22 +17,22 @@ type Claims struct {
 }
 
 type TokenManager struct {
-	TokensToUpdate []uint64 // UserID
+	TokensToUpdate []*pb.UpdateTokenRecord // UserID
 }
 
 // TODO: probably most of these methods (and struct fields) can be changed to unexported
 
 // JWTUpdateRequired returns element index if the user ID is in the list,
 // otherwise returns -1 to indicate that JWT update is not needed for this user
-func (tm *TokenManager) JWTUpdateRequired(claims *Claims) uint64 {
-	index := -1
-	for i, tokenID := range tm.TokensToUpdate {
-		if claims.UserID == tokenID {
+func (tm *TokenManager) JWTUpdateRequired(claims *Claims) int {
```

Comment:
I would prefer to avoid the `JWT` prefix as well as unexporting them (if not needed outside the package).

Also, I think this method should return a `bool` instead since the caller can easily find the claims's `UserID` via `claims.UserID`.

Reply:
Done.